packages/@okta/i18n/src/properties/login.properties

```diff
@@ -14,12 +14,14 @@ autoPush = Send push automatically
 unlockaccount = Unlock account?
 haveaccount = Already have an account ?
 needhelp = Need help signing in?
-goback = Back to Sign In
 forgotpassword = Forgot password?
 help = Help
 retry = Retry
 iphone = iPhone
 android = Android
+## used in v1
+goback = Back to Sign In
```

Comment:
Can we reuse the V1 key `goback` to preserve any string customization?

Reply:
Chatted with @vincentvoong-okta that `Back to sign in` seems more right hence would have little impact for V1.
I'll change it separately to `main` branch.